File: app/models/dish.rb
class Dish < ApplicationRecord
    # :id, :name, :cuisine, :price, :image_url, :restaurant_name, :city_name
<<<<<<< HEAD
    has_many :users, dependent: :destroy
    has_many :reviews, through: :users
    
=======
    has_many :reviews, dependent: :destroy
    has_many :users, through: :reviews
>>>>>>> 80fddbf2801f3dbc38a536304dd02189ba167df2
    validates :name, presence: true
    validates :cuisine, presence: true
    validates :price, numericality: true
    validates :restaurant_name, presence: true
    validates :image_url, presence: true
    validates :city_name, presence: true
end

File: client/src/components/Dish.js
import React from "react";

function Dish ({ id, name, cuisine, price, image_url, restaurant_name, city_name, reviews, users, setUsers }) {

    return (
        <div>
            <h2>{name}</h2>
            <h2>{restaurant_name}</h2>
            <h3>${price}</h3>
            <h4>{cuisine}</h4>
            <img src={image_url} />
            <h5>Location: {city_name}</h5>
            <p>{reviews}</p>
            
        </div>
    )
}
export default Dish;